### PR TITLE
Upgrade To IHP V20

### PR DESCRIPTION
**Pull request by s0kil**
Adjust `default.nix` ref when IHP 20 release is created.

### CODE REVIEW

File: Web/View/Comments/Index.hs [hunk 1]
```diff
@@ -26,8 +26,8 @@ instance View IndexView where
 renderComment comment = [hsx|
     <tr>
         <td>{comment}</td>
-        <td><a href={ShowCommentAction (get #id comment)}>Show</a></td>
-        <td><a href={EditCommentAction (get #id comment)} class="text-muted">edit</a></td>
-        <td><a href={DeleteCommentAction (get #id comment)} class="js-delete text-muted">Delete</a></td>
+        <td><a href={ShowCommentAction (comment.id)}>Show</a></td>
```
Comment (mpscholten) [inline Web/View/Comments/Index.hs]: I think we can remove the quotes here (and at some other places):

```suggestion
        <td><a href={ShowCommentAction comment.id}>Show</a></td>
```

File: default.nix [hunk 2]
```diff
@@ -1,6 +1,7 @@
 let
     ihp = builtins.fetchGit {
         url = "https://github.com/digitallyinduced/ihp.git";
+        ref = "master";
```
Comment (s0kil) [inline default.nix]: For some reason, on different machine, Nix cannot resolve rev, maybe different Nix version installed.

**Comment by s0kil** [comment]
Ready to go.

**Review by mpscholten — APPROVED** [review]
Thanks!

**Pull request merged**